Source/WebCore/platform/Supplementable.h

```diff
@@ -122,7 +122,7 @@ class Supplementable {
 #endif
 
 private:
-    using SupplementMap = HashMap<ASCIILiteral, std::unique_ptr<Supplement<T>>, ASCIILiteralPtrHash>;
+    using SupplementMap = HashMap<ASCIILiteral, std::unique_ptr<Supplement<T>>>;
```

Comment:
We may be able to get rid of the ASCIILiteralPtrHash implementation too.

Reply:
Hm... it's still used in NetworkProcessSupplementMap, WebContextSupplementMap, and WebProcessSupplementMap. I guess we could move them off of this hash traits as well?

Reply:
Yes, we should. They all are error prone in the same way.

Reply:
Let me do that in a separate PR though.

Reply:
https://github.com/WebKit/WebKit/pull/27059